modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -37,6 +44,11 @@ For {ubuntu} 18.04:
 * From the {webui}, add [systemitem]``Ubuntu 18.04`` and [systemitem]``SUSE Linux Enterprise Client Tools Ubuntu 1804 amd64``.
 * From the command prompt, add [systemitem]``ubuntu-18.04-pool-amd64`` and [systemitem]``ubuntu-18.04-suse-manager-tools-amd64``.
 
+For {ubuntu} 20.04:
+
+* From the {webui}, add [systemitem]``Ubuntu 20.04`` and [systemitem]``SUSE Linux Enterprise Client Tools Ubuntu 2004 amd64``.
+* From the command prompt, add [systemitem]``ubuntu-20.04-pool-amd64`` and [systemitem]``ubuntu-20.04-suse-manager-tools-amd64``.
+
 [NOTE]
 ====
 The mandatory channels do not contain {ubuntu} upstream packages.
```

Comment:
This is not correct anymore.

Reply:
![image](https://user-images.githubusercontent.com/3914967/83858447-dece1180-a75f-11ea-892c-987addd80317.png)

Reply:
Click on the "bars" for "Ubuntu 20.04". You will find the channels there.

Reply:
So it needs to be all three of the Mandatory ones here? 
![image](https://user-images.githubusercontent.com/3914967/83859523-65372300-a761-11ea-93c6-6addc90f185a.png)

Reply:
Yes, but as you can see on the WebUI, you can't pick them one by one (you check the product, and that's it).